modules/userInterface.js

```diff
@@ -0,0 +1,13 @@
+function addNewhandleClick() {
```

Comment:
- [ ] Kindly 🙏🏻 use the **ES6** `arrow` functions over the `ES5` functions, while ES5 remains widely supported but using ES6 features can enhance your productivity, improve code readability, and make your codebase more maintainable. Even in today's programming era, there are lots of features introduced by the `ECMAScript` or `ES` community which are not available in old *ES5* standards. I mentioned in my previous comment regarding the following project requirements. Kindly adhere to the project requirements and implement the requested changes.

### Requirement

![Screenshot from 2023-07-10 16-37-51](https://github.com/agneta1991/Awesome-books-with-ES6/assets/66476329/2dfbfd0d-1aa1-4261-93b3-a7f5222f9410)